guides/mentorship/README.md

```diff
@@ -0,0 +1,33 @@
+#Made Tech Mentorship Guide
+
+##Overview
+At Made Tech, mentorship enables Made Tech staff with the opportunity to provide guidance and support to people in the wider tech community. Mentees are able to gain new knowledge and support and develop and enhance their skills.
+
+###General Guidance
+* Be kind, have empathy and be supportive
+* Don’t share personal information about your mentee in open forums
+* Please give at least 48 hours notice before moving or cancelling your session, so that both parties have a chance to adjust their schedules
+* When first meeting, take some time to outline what goals you’d like to achieve, and how long it might take to achieve them. We recommend setting somewhere between 6 and 12 sessions and reviewing at the end to see where to go from there. 
+
+###Mentee / Mentor Matching
+Once you're matched up, we'll send an introductory email introducing you to your mentor.
+
+###First session guidance
+If you have a particular approach or mentorship / pairing style that you’d like to, please feel free to use it. Here are some suggestions to get the ball rolling on the first session:
+1. Introduce yourselves
+2. Outline goals and objectives
+3. Discuss what you both would like out of the sessions - Set some SMART Goals
```

Comment:
Would it be good for an explanation of SMART Goals here? Perhaps as a separate page, footnote, etc?